components/ui/people-card.tsx

```diff
@@ -6,39 +6,118 @@ import Image from "next/image";
 import { Card, CardContent, CardTitle, CardDescription } from "@/components/ui/card";
 import { cn } from "@/lib/utils";
 import { cardVariants } from "@/components/ui/variants";
+import * as Dialog from "@radix-ui/react-dialog";
+import { FaGithub, FaInfoCircle } from "react-icons/fa";
 
 interface CardWithImageProps {
   className?: string;
   imagePath: string;
   hoverImagePath?: string;
   name: string;
   title: string;
+  personDetails?: {
+    name: string;
+    title: string;
+    team?: string;
+    subteam?: string;
+    type?: string;
+    github_username?: string;
+    brown_directory_uuid?: string;
+    bio?: string;
+    image?: string;
+  };
 }
 
-export const CardWithImage: React.FC<CardWithImageProps> = ({ className, imagePath, hoverImagePath, name, title, ...props }) => {
+export const CardWithImage: React.FC<CardWithImageProps> = ({ className, imagePath, hoverImagePath, name, title, personDetails, ...props }) => {
   const [isHovered, setIsHovered] = useState(false);
+  const [open, setOpen] = useState(false);
 
   return (
-    <Card
-      className={cn("overflow-hidden", cardVariants({ variant: "people" }), "w-[400px]", "h-[600px]", "flex-shrink-0")}
-      onMouseEnter={() => setIsHovered(true)}
-      onMouseLeave={() => setIsHovered(false)}
-    >
-      <CardContent className="flex flex-col h-full">
-        <div className="relative overflow-hidden">
-          <Image
-            src={isHovered && hoverImagePath ? hoverImagePath : imagePath}
-            alt={name}
-            width="500"
-            height="500"
-            className="rounded-full transition-opacity duration-300 max-h-[350px] min-h-[350px] max-w-[350px]"
-          />
-        </div>
-        <div>
-          <CardTitle className="text-2xl text-center py-4">{name}</CardTitle>
-          <CardDescription className="text-xl italic text-center">{title}</CardDescription>
-        </div>
-      </CardContent>
-    </Card>
+    <Dialog.Root open={open} onOpenChange={setOpen}>
+      <Dialog.Trigger asChild>
+        <Card
+          className={cn("overflow-hidden cursor-pointer", cardVariants({ variant: "people" }), "w-[400px]", "h-[600px]", "flex-shrink-0")}
+          onMouseEnter={() => setIsHovered(true)}
+          onMouseLeave={() => setIsHovered(false)}
+          onClick={() => setOpen(true)}
+        >
+          <CardContent className="flex flex-col h-full">
+            <div className="relative overflow-hidden">
+              <Image
```

Comment:
The ones that have the CCV logo... it looks like an oval. Is the file not a square?

![Screenshot 2025-07-02 at 1 42 05 PM](https://github.com/user-attachments/assets/829cb5b4-79a9-4804-872d-e30ae10eaaa0)